Woof/Woof/View/ReusableViews/ReusableLabelView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+/// A view displaying a section for the detailed sitter's information.
+struct ReusableLabelView: View {
+    /// The name of an icon.
+    let image: String
```

Comment:
The name might be confusing. This is not an image, but the name of image. Moreover, image of the icon:

```suggestion
    /// The name of the icon that the label displays.
    let iconName: String
```

Reply:
Thanks, fixed it.